src/services/jwt.service.js

```diff
@@ -0,0 +1,34 @@
+import jwt from 'jsonwebtoken';
+
+function sign(user) {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '30s'
```

Comment:
The access token expiration time is set to '30s', which is quite short for most applications. Consider increasing this duration to a more practical value, such as '15m' or '1h', depending on your application's requirements.